ext/SciMLBaseZygoteExt.jl

```diff
@@ -40,32 +40,36 @@ import SciMLStructures
     VA[i, j], ODESolution_getindex_pullback
 end
 
-@adjoint function Base.getindex(VA::ODESolution, sym, j::Int)
-    function ODESolution_getindex_pullback(Δ)
-        i = symbolic_type(sym) != NotSymbolic() ? variable_index(VA, sym) : sym
-        du, dprob = if i === nothing
-            getter = getobserved(VA)
-            grz = pullback(getter, sym, VA.u[j], VA.prob.p, VA.t[j])[2](Δ)
-            du = [k == j ? grz[2] : zero(VA.u[1]) for k in 1:length(VA.u)]
-            dp = grz[3] # pullback for p
-            dprob = remake(VA.prob, p = dp)
-            du, dprob
-        else
```

Comment:
Is this because the ChainRulesCore one is fixed? If so, just delete instead of comment.

Reply:
Oops, yeah.